index.js

```diff
@@ -57,6 +57,10 @@ Benchmarked.prototype.defaults = function(benchmarked) {
     }
   };
 
+  this.format = function(benchmark) {
+    return '  ' + benchmark;
+  };
```

Comment:
agreed. What I wrote only works when providing the option, which I'm always doing.

I admit I was confused by the Benchmarked constructor and the Base. I missed the part where the options are provided to the Base first and then available in the defaults function as `this.options`. I threw in those lines and it worked. I didn't test it properly.

The defaults function still seems like the right place to handle it because it's an options thing.

I'll make a new PR with the change from @doowb and with some added tests.